pkg/manifests/manifests.go

```diff
@@ -79,17 +79,15 @@ func (f *Factory) RouterDaemonSet(cr *ingressv1alpha1.ClusterIngress) (*appsv1.D
 
 	ds.Name = name
 
-	ds.Spec.Template.Labels = map[string]string{
-		"app":    "router",
-		"router": name,
+	if ds.Spec.Template.Labels == nil {
+		ds.Spec.Template.Labels = map[string]string{}
```

Comment:
`app` not required?  in here and the other maps below.

Reply:
Yes, `app` label is already defaulted in the yaml file:
https://github.com/openshift/cluster-ingress-operator/blob/f0ab7cc3c0e9b6cda2b565f1acdfd0456ffb505c/assets/router/daemonset.yaml#L17